### PR TITLE
sidebar: calc panels layout

### PR DESCRIPTION
**Pull request by eszkadev**
patch2:

    sidebar: css: calc area, line, shadow, possize panels
    context: shape

patch1:

    sidebar: css: calc number format and alignment panels
    context: default/cell
    - removed not needed direct position rules

### CODE REVIEW

**Comment by eszkadev** [comment]
what this fixes, examples:

patch1

File: browser/css/jssidebar.css [hunk 1]
```diff
@@ -318,17 +307,6 @@ td.jsdialog .jsdialog.cell.sidebar {
...
-	display: none;
-}
-
-#fillattrhb {
-	margin-left: 4px;
-	position: relative;
-	left: 38px;
-}
-
-
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:331]: :+1:  fixes misalignments image

File: browser/css/jssidebar.css [hunk 2]
```diff
@@ -193,11 +187,6 @@ div#thousandseparator.checkbutton.jsdialog.sidebar {
 #NumberFormatPropertyPanel > tr > td > div > div > div > div > div > div {
 	text-align: right;
 }
-#NumberFormatPropertyPanel #table-box2,
-#NumberFormatPropertyPanel #leadingzeroes {
-	position: relative;
-	left: 9px;
-}
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:200]: :+1: Fixed misalignment: 
image

File: browser/css/jssidebar.css [hunk 3]
```diff
@@ -367,49 +345,16 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
...
-
-.sidebar #fillgrad1 {
-	position: relative;
-	left: 7px;
-}
-.sidebar #fillgrad2,
-.sidebar #LB_SHADOW_COLOR {
-	position: relative;
-	left: 5px;
-}
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:389]: :+1:

File: browser/css/jssidebar.css [hunk 4]
```diff
@@ -367,49 +345,16 @@ button#button2.ui-pushbutton.jsdialog.sidebar {
...
-
-.sidebar #leftindent,
-.sidebar #LB_GLOW_TRANSPARENCY,
-.sidebar #LB_DISTANCE,
-.sidebar #LB_SHADOW_BLUR {
-	position: relative;
-	right: 18px;
-}
-
```
Comment (pedropintosilva) [inline browser/css/jssidebar.css:412]: :+1:

**Review by pedropintosilva — APPROVED** [review]

**Pull request merged**